sync/api/projects.py

```diff
@@ -269,3 +281,52 @@ def create_project_submission_with_eventlog_bytes(
         return Response(error=SubmissionError(message="Failed to upload event log"))
 
     return Response(result=response["result"]["submission_id"])
+
+
+def create_project_recommendation(project_id: str, **kwargs) -> Response[str]:
```

Comment:
Can we be more specific for the developer API? There's only a couple valid options going in to kwargs here - let's call them out.

Reply:
If we don't want to advertise the options let's just go with `**options`

Reply:
Yea sorry, the idea was to not advertise the options. Happy to rename to **options.